src/VSIX/VSIX.csproj

```diff
@@ -114,13 +117,26 @@
     <Reference Include="System.Windows.Forms" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.2.32505.173" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VisualStudio.Data.Framework" Version="17.2.32505.113" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.2.2186" />
+    <PackageReference Include="Microsoft.VisualStudio.Data.Framework">
+      <Version>15.9.28307</Version>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.SDK">
+      <Version>15.0.1</Version>
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VSSDK.BuildTools">
+      <Version>17.1.4057</Version>
```

Comment:
Just making sure it's OK to mix major versions like this, between e.g. Microsoft.VisualStudio.SDK and Microsoft.VSSDK.BuildTools (I have no idea - just asking)

Reply:
Sure thing, if it works, it works...